### PR TITLE
docs: Dark mode styling fix

### PR DESCRIPTION
**Pull request by matthewlipski**

### CODE REVIEW

File: packages/website/docs/demoUtils.ts [hunk 1]
```diff
@@ -14,5 +14,16 @@ export const getStyles = (isDark: Ref<boolean>): string => `body {
   -webkit-tap-highlight-color: transparent;
   -webkit-touch-callout: none;
   
-  background-color: ${isDark ? "#151515" : "white"};
+  background-color: ${isDark ? "#151515" : "white"} !important;
 }`;
+
+export const index = `import React, { StrictMode } from "react";
```
Comment (YousefED) [inline packages/website/docs/demoUtils.ts:20]: why was this added / needed?

Comment (matthewlipski) [reply packages/website/docs/demoUtils.ts:20]: Forgot to remove

Comment (matthewlipski) [reply packages/website/docs/demoUtils.ts:20]: Forgot to remove

Comment (YousefED) [reply packages/website/docs/demoUtils.ts:20]: ok, so will be fixed / removed?
